forge/context-driver/sequelize.js

```diff
@@ -58,6 +60,47 @@ module.exports = {
             type: Sequelize.Transaction.TYPES.IMMEDIATE
         },
         async (t) => {
+            const quotaLimit = app.config?.context?.quota || 0
```

Comment:
@hardillb Do we need a safe default here?

Reply:
We need an option to have no limits

Reply:
OK, so if we leave this as is, then that is catered for already (line below checks that `quotaLimit > 0` before doing any checks